libraries/android/castor-tokens/version.json

```diff
@@ -0,0 +1,4 @@
+{
+  "versionCode": 0,
+  "versionName": "0.0.0"
```

Comment:
you can set `0.1.0` for versionName till you have a version you would like to release according to semantic version

Reply:
This is only a placeholder, we have it scripted to bump alongside npm package.

After merging I am planning to release this as `1.0.0-beta.4` to follow [existing release structure](https://github.com/onfido/castor-tokens/releases), is this valid for Android, or how do you identify a beta release?

Reply:
`1.0.0-beta.4` valid string for versionName 

versionName field -> string
versionCode -> int

Reply:
Great.

I did that version code is only increased with every release. I saw some do that they convert actual version to an integer, let's say `1.0.4` would become `104` or something, however I think for us just increasing by 1 each time will be fine.